sales.html

```diff
@@ -2,11 +2,26 @@
 <html>
 <head>
   <title>Store Sales Summary Page</title>
+  <link rel="stylesheet" href="reset.css">
   <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="salesstyle.css">
 </head>
 <body>
+  <header>
+    <div class="restriction_div">
+      <a href="index.html">
+        <img src="images/salmon.jpg">
+      </a>
+      <ul>
+        <li>Buy Cookies!</li>
+        <li>Locations</li>
+        <li>Our Process</li>
```

Comment:
These guys should be anchor tags with "#" as the href. They don't have to go somewhere in order to be links.